### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by amartya-rs**
Product listing page of ecom site.

### CODE REVIEW

File: pages/product-page/products.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+}
+
+.grid-3-column {
+    grid-template-columns: repeat(auto-fit, minmax(17rem, 1fr));
+}
+
+
+/*list styling*/
+li {
+    margin-bottom: 10px ;
```
Comment (rutu14) [inline pages/product-page/products.css:38]: Use a selector, so that other li elements are not styled in the same way.

**Review by rutu14 — APPROVED** [review]

**Review by bhushannaikaroskar — APPROVED** [review]
Looks good to merge

File: pages/product-page/products.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+.main-nav {
+    grid-area: nav;
+}
+
+aside {
+    background: var(--WHITE);
+    grid-area: aside;
+}
+
+aside div{
```
Comment (dev-prabhat) [inline pages/product-page/products.css]: try to use class to apply style

**Review by dev-prabhat — APPROVED** [review]
Looks good to me

File: pages/product-page/products.css (hunk 1 above)
Comment (amartya-rs) [reply pages/product-page/products.css:38]: Made the changes.

File: pages/product-page/products.css (hunk 2 above)
Comment (amartya-rs) [reply pages/product-page/products.css]: Made the changes.

**Pull request merged**